src/core/storage/providers/local/local_file_entity.ts

```diff
@@ -31,7 +33,7 @@ export class LocalFileEntity implements FileEntity{
     async upload(body: NodeJS.ReadableStream): Promise<FileEntity> {
         
         // Get the directory from file path
-        const fullUploadPath = this.rootUploadPath+this.filePath;
+        const fullUploadPath = this.serverRoot+this.rootUploadPath+this.filePath;
```

Comment:
suggest using nodejs path libraries, rather than these custom code.
fullUploadPath =path.join(this.serverRoot, this.rootUploadPath, this.filePath)

Reply:
this is fixed